### DISCUSSION TITLE
restarting addon from automation in Home Assistant

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [mrlt8/docker-wyze-bridge#1028] [Q&A]
Is there a command I can use in a Home Assistant automation to restart the docker-wyze-bridge addon? Sometimes one of my wyze cam v3 drop from the addon. When I restart the addon, it reappears.

**Comment by terryhonn** [comment]
Occasionally one or more of my cameras will stop being seen in Frigate, so I made a shell command to call the wyze-bridge API camera restart endpoint. It's not a complete restart of the bridge, but using this, I find that I seldom need to fully restart wyze-bridge:
```

shell_command:
   restart_wyze_cams: 'curl http://192.168.1.102:5001/restart/cameras'

```

**Reply by Cold-Lemonade** [reply]
@terryhonn -- Thank you. Is "192.168.102:5001" the ip address & port number of your docker-wyze-bridge instance? I am just trying to figure out what to point that curl command to on my set up.

**Reply by terryhonn** [reply]
That's right, sorry I didn't note that.

**Comment by Cold-Lemonade** [comment]
Does anyone know if the following YAML code will work in Home Assistant to restart the docker-wyze-bridge addon?

```
- service: hassio.addon_restart
  data:
    addon: "7094bb28_docker-wyze-bridge"
```
